optional_requirements.txt

```diff
@@ -4,3 +4,4 @@ SimpleITK; python_version != '3.4' and python_version != '3.5'
 astropy
 tifffile
 imageio
+PyWavelet
```

Comment:
I don't think it's an optional requirement. Should go in requirements.txt. That's why travis didn't install it.